### PR TITLE
Remove unused properties

### PR DESCRIPTION
**Pull request by crazytonyli**
No logic is changed, just organise the property declarations by their access scope and remove a couple of unused properties.

### CODE REVIEW

File: ParselyTracker/ParselyTracker.swift [hunk 1]
```diff
@@ -26,17 +14,29 @@ public class Parsely {
...
     lazy var visitorManager = VisitorManager()
+    internal static func getInstance() -> Parsely {
+        return Parsely()
+    }
+
+    private var _track: Track!
+    private var configured = false
+    private var flushTimer: Timer?
+    private var flushInterval: TimeInterval = 30
+    private var backgroundFlushTask: UIBackgroundTaskIdentifier = UIBackgroundTaskIdentifier.invalid
```
Comment (mokagio) [inline ParselyTracker/ParselyTracker.swift]: Nitpick

```suggestion
    private var backgroundFlushTask = UIBackgroundTaskIdentifier.invalid
```

or

```suggestion
    private var backgroundFlushTask: UIBackgroundTaskIdentifier = .invalid
```

**Review by mokagio — APPROVED** [review]
I run the tests locally and they passed

**Comment by crazytonyli** [comment]
Not sure why this PR didn't trigger the GitHub Action...

**Comment by mokagio** [comment]
> Not sure why this PR didn't trigger the GitHub Action...

I was wondering that too in another PR. I just thought of looking at the GitHub Action config. The run only on PR against `master` 🤦‍♂️ : 

https://github.com/Automattic/AnalyticsSDK-iOS/blob/6f0ab679f0de687e09814d75acac7aee75e48e96/.github/workflows/ios.yml#L3-L7

**Comment by crazytonyli** [comment]
@mokagio Good find! Well, that's one solid reason to just use the `master` branch, instead of a custom branch, then.

**Pull request merged**